doc/guides/device_mgmt/ota.rst

```diff
@@ -0,0 +1,65 @@
+.. _ota:
+
+Over-the-Air Update
+###################
+
+Overview
+********
+
+Over-the-Air (OTA) Update is a method for delivering firmware updates to remote
+devices using a network connection. Although the name implies a wireless
+connection, devices receiving updates over a wired connection (such as Ethernet)
+are still commonly referred to by OTA. This approach requires server
+infrastructure to host the firmware binary and implement a method of signaling
+when an update is available.
+
+The :ref:`dfu` section discusses upgrading Zephyr firmware using MCUboot. The
+same method can be used as part of OTA. The binary is first downloaded
+into an unoccupied code partition, then flashed using the `MCUboot bootloader`_
+process.
+
+Examples of OTA
+***************
+
+Golioth
+=======
+
+`Golioth`_ is an IoT management platform that includes Zephyr OTA updates.
+Devices are configured to observe your available firmware revisions on the
+Golioth Cloud. When a new version is available, the device downloads and flashes
+the binary.
+
+Security is a concern with OTA updates. In this implementation, the connection
+between cloud and device is secured using TLS/DTLS, and the signed firmware binary
+is confirmed by MCUboot before flashing occurs.
```

Comment:
Security is mentioned here but not anywhere else. For parallelism, I think it makes sense to:

- move the blanket statement that security is a concern higher up, outside of the sections for any individual solution
- document the security related bits of the implementation for each solution

Reply:
Will do.